analysis/src/CompletionBackEnd.ml

```diff
@@ -1817,6 +1942,26 @@ Note: The `@react.component` decorator requires the react-jsx config to be set i
            in
            (dec2, doc))
     |> List.map mkDecorator
+  | Cargument {functionContextPath; argumentLabel; prefix} -> (
+    let labels =
+      match
+        functionContextPath
+        |> getCompletionsForContextPath ~full ~opens ~rawOpens ~allFiles ~pos
+             ~env ~exact:true ~scope
+        |> completionsGetTypeEnv
+      with
+      | Some (typ, _env) -> typ |> getArgs ~full ~env
+      | None -> []
+    in
+    let targetLabel =
+      labels |> List.find_opt (fun (label, _) -> label = argumentLabel)
+    in
+    match targetLabel with
+    | None -> []
+    | Some (Labelled _, typ) ->
+      typ |> completeTypedValue ~env ~full ~prefix ~expandOption:true
```

Comment:
Why expand option on every labeled arg?

Reply:
Trying to capture that you don't need to qualify optional values, but that's only for optional arguments, right? So this should be for optional, not labeled, arguments

Reply:
Fixed in a recent commit.